### DISCUSSION TITLE
Python binding

### DISCUSSION THREAD

**Discussion by jlb6907** [slint-ui/slint#2056] [General] [answered]
Hi, are you planning to provide an official binding for the python language ?

**Comment by hunger** [comment]
We would like to, but we have too many other tasks with a higher priority right now. I do not expect us to get round to it in the next month/year.

But contributions would be welcome:-)

**Reply by melMass** [reply]
Hi,

Is there a guide or some pointers on how to create such bindings? I'm not sure what it involves but I guess the JS one could serve as a guideline. Now that I understand Slint better I really think python would be an interesting target

**Reply by hunger** [reply]
You will probably want to use the Slint interpreter with python, just like the JS API does. So yes, that can be one source of inspiration.

I think the easiest is to just join chat.slint.dev and just start to ask questions there. That's more direct than github discussions.

**Comment by ogoffart** [comment]
Yes we plan to support more languages, including Python. But as @hunger point out, it is currently not in our short term priority list.

**Comment by aurindam** [comment, marked as answer]
@jlb6907  Support for Python has been kicked off with an initial PR - https://github.com/slint-ui/slint/pull/4155. Track progress using - https://github.com/slint-ui/slint/milestone/18.
